### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by friedcucumber**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128641072/21.1.5%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E4288)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-# Display Theme Selectors Based on BarItems and Hide Themes from List
+# WPF MVVM Behaviors - Display Theme Selectors Based on BarItems and Hide Themes from List
 
 DevExpress WPF Controls include the following behaviors that display a theme gallery in [Bars](https://docs.devexpress.com/WPF/DevExpress.Xpf.Bars.Bar) and [Ribbon](https://docs.devexpress.com/WPF/DevExpress.Xpf.Ribbon.RibbonControl):
```
Comment (dirk-pieterse) [inline Readme.md]: `in [Bars] and [Ribbon]` Bars and ribbon what? This sentence is incomplete

File: Readme.md [hunk 2]
```diff
@@ -23,6 +22,13 @@ You can use the [Theme.ShowInThemeSelector](https://docs.devexpress.com/WPF/DevE
 
 In this sample, the [Standard](https://docs.devexpress.com/WPF/DevExpress.Xpf.Core.Theme.StandardCategory) and [Metropolis](https://docs.devexpress.com/WPF/DevExpress.Xpf.Core.Theme.MetropolisCategory) theme categories are hidden from theme selectors.
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
In this example, the [Standard](https://docs.devexpress.com/WPF/DevExpress.Xpf.Core.Theme.StandardCategory) and [Metropolis](https://docs.devexpress.com/WPF/DevExpress.Xpf.Core.Theme.MetropolisCategory) theme categories are hidden.
```

**Review by dirk-pieterse — APPROVED** [review]

File: Readme.md (hunk 1 above)
Comment (friedcucumber) [reply Readme.md]: in [Bars] and [Ribbon] controls

Comment (friedcucumber) [inline Readme.md]:
```suggestion
DevExpress WPF Controls include the following behaviors that display a theme gallery in [Bars](https://docs.devexpress.com/WPF/DevExpress.Xpf.Bars.Bar) and [Ribbon](https://docs.devexpress.com/WPF/DevExpress.Xpf.Ribbon.RibbonControl) controls:
```

Comment (friedcucumber) [reply Readme.md]: Are you agree with that version?

Comment (dirk-pieterse) [reply Readme.md]: Yes, I agree with this version

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**